### PR TITLE
frontend structure is complete

### PR DESCRIPTION
**Pull request by yoshihiroyamada23**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/catindex" element={<CatIndex />} />
+      <Route path="/catshow" element={<CatShow />} />
+      <Route path="/catnew" element={<CatNew />} />
+      <Route path="/catedit" element={<CatEdit />} />
+      <Route path="*" element={<NotFound />} />
+      </Routes>
+      < Footer />
+  </>
```
Comment (sjproctor) [inline src/App.js:29]: Fix the indentation here please!

**Review by sjproctor — COMMENTED** [review]
Fix your indentation please! Nested tags (tags inside other tags) need to be indented.

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,45 @@
-.App {
+*{background-color: blue;}
```
Comment (elysemontano) [inline src/App.css]: Let's fix the indentation so it is consistent with the rest of your css styles.

File: src/Components/Header.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const Header = () => {
+return (
+    <div className="header">
+        <header>
+        </header>
```
Comment (elysemontano) [inline src/Components/Header.js]: I would recommend adding some content in here such as a navbar.

File: src/Pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import HomeImage from '../Assets/HomeImage.png'
+
+const Home = () => {
+return (
+    <div className="home">
+    <h1>Welcome to Cat Tinder</h1>
+    <img src={HomeImage} alt ="cat" className="logo"/>
+    </div>
```
Comment (elysemontano) [inline src/Pages/Home.js]: Check indentation

File: src/Pages/NotFound.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import FooterImage from '../Assets/FooterImage.png'
+
+const NotFound = () => {
+return (
+    <>
+   <header>
+    <img src={FooterImage} alt="Footer" className="nopage"/>
+   </header>
+    </>
```
Comment (elysemontano) [inline src/Pages/NotFound.js]: Check indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left some comments in App.css, NotFound, Home, and Header

**Review by elysemontano — COMMENTED** [review]
I would recommend updating the Header component to also include navigation. Currently your user will not be able to move about from the home page unless they use the URL.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**